projects/1.16/assets/phi/phi/lang/zh_cn.json

```
@@ -0,0 +1,257 @@
+{
+	"phi.spellparam.frequency": "频率",
+	"phi.spellparam.from": "从",
+	"phi.spellparam.to": "到",
+	"phi.spellparam.from_to": "从/到",
+	"phi.spellparam.to_from": "从/到（反向）",
+	"phi.spellparam.condition": "状态",
+	"phi.spellparam.positive": "正",
+	"phi.spellparam.negative": "负",
+	"phi.spellparam.div": "A / B",
+	"phi.spellparam.mod": "A %% B",
+	"phi.spellparam.digit": "位",
+	"phi.spellparam.target1": "目标A",
+	"phi.spellparam.target2": "目标B",
+	"phi.spellparam.prefix": "Prefix",
+	"phi.spellparam.text": "文本",
+	"phi.spellparam.text1": "文本 A",
+	"phi.spellparam.text2": "文本 B",
+	"phi.spellparam.text3": "文本 C",
+	"phi.spellparam.text4": "文本 D",
+	"phi.spellparam.before": "在。。。前",
+	"phi.spellparam.after": "在。。后",
+	"phi.spellparam.at": "At",
+	"phi.spellparam.default": "默认",
+	"phi.spellparam.line": "行",
+
+	"phi.spellerror.range": "错误: 此术式的范围为32方块",
```

```suggestion
	"phi.spellerror.range": "错误：此术式的范围为32方块",
```
太多了，基本都是半角`: `
用正则批量替换吧